frontend/src/container/SideNav/NavItem/NavItem.styles.scss

```diff
@@ -75,6 +75,10 @@
 			text-overflow: ellipsis;
 		}
 	}
+
+	.beta-tag {
+		padding-right: 0;
+	}
```

Comment:
Added this as unnecessary `padding` and `margin` were there, and for longer texts like - `Messaging Queues` its was not looking good